### PR TITLE
Upgrade tooling

### PR DESCRIPTION
**Pull request by poteirard**
- Upgrade react scripts
- Upgrade rollup
- Change browserslist

### CODE REVIEW

File: etc/semantic-release/config.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 module.exports = {
   dryRun: false,
   plugins: [
-    '@semantic-release/commit-analyzer',
+    [
+      '@semantic-release/commit-analyzer',
+      {
+        preset: 'conventionalcommits',
```
Comment (poteirard) [inline etc/semantic-release/config.js:7]: by default it's `angular` but we use `conventionalcommits`

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "@types/util-deprecate": "^1.0.0",
     "core-js": "^3.3.5",
     "downshift": "^5.0.0",
+    "jest": "24.9.0",
```
Comment (admmasters) [inline package.json:40]: Can we go to 25? They are about to release 26 now..

Comment (poteirard) [reply package.json:40]: react-scripts force you to have that version

**Pull request merged**